syntax = "proto3";

package example;

message TaskRequest {
    int32 num_workers = 1;
    string leader_uuid = 2;
}
